### PR TITLE
DOCS-16652 Adds Reversing State

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Adds `REVERSING` to table of mongosync states.

## Staging

* [mongosync States](https://preview-mongodbkennethdyer.gatsbyjs.io/cluster-sync/DOCS-16652-mongosync-states/reference/mongosync-states/#state-descriptions)

## JIRA

[DOCS-16652](https://jira.mongodb.org/browse/DOCS-16652)

## Build

* [2024-03-05](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65e75b5311db7b0427bebccd)
* [2024-03-13](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65f1ceee6fc5f048501f7b86)

### CODE REVIEW

File: source/reference/mongosync-states.txt [hunk 1]
```diff
@@ -85,3 +85,15 @@ operations in that state.
...
 
+   * - .. _c2c-state-reversing:
+
+       ``REVERSING``
+
+     - The sync process copies metadata from the destination
+       cluster to the source cluster. When complete, the phase
+       reverts to applying change events and the original source
+       cluster becomes the new destination cluster while the old
+       destination becomes the new source.
```
Comment (nvillahermosa-mdb) [inline source/reference/mongosync-states.txt:96]: Not sure about the use of "revert" here. Would it be accurate to say something like:

```suggestion
     - The sync process copies metadata from the destination
       cluster to the source cluster. Then, MongoDB swaps the
       source and destination clusters and resumes applying
       change events.
```

I don't love the use of "swap" in my example from a technical standpoint, but if possible I'd prefer a shorter way of saying that source and destination switch places.

**Review by nvillahermosa-mdb — CHANGES_REQUESTED** [review]
Thanks, Kenneth! I have some wording questions as far as clarity.

**Comment by kennethdyer** [comment]
@nickkenney-mdb, Changes applied.  Anything else you'd like to see?

**Review by huan-Mongo — CHANGES_REQUESTED** [review]
Screenshot 2024-03-08 at 9 35 39 AM

Hi @kennethdyer Somehow the states table on the stage page disappears. Could you take a look? Thx.

**Comment by kennethdyer** [comment]
@huan-Mongo, I'm not sure what's happening on your end.  When I'm running it in Chrome on Mac it renders correctly.  I have noticed that tables sometimes take a beat to load, but I think that's a platform issue.

File: source/reference/mongosync-states.txt [hunk 2]
```diff
@@ -85,3 +85,14 @@ operations in that state.
      - - ``GET`` :ref:`/progress <c2c-api-progress>`
        - ``GET`` :ref:`/reverse <c2c-api-reverse>`
```
Comment (huan-Mongo) [inline source/reference/mongosync-states.txt:86]:
```suggestion
       - ``POST`` :ref:`/reverse <c2c-api-reverse>`
```
reverse is a `POST` command.

File: source/reference/mongosync-states.txt [hunk 3]
```diff
@@ -85,3 +85,14 @@ operations in that state.
...
+   * - .. _c2c-state-reversing:
+
+       ``REVERSING``
+
+     - The sync process copies metadata from the destination
+       cluster to the source cluster. Then, MongoDB swaps the
+       source and destination clusters and resumes applying
+       change events.
+     - - ``POST`` :ref:`/reverse <c2c-api-reverse>`
+       - ``GET`` :ref:`/progress <c2c-api-progress>`
```
Comment (huan-Mongo) [inline source/reference/mongosync-states.txt:97]:
```suggestion
     - - ``GET`` :ref:`/progress <c2c-api-progress>`
```

I just noticed `reverse` is not a possible command under this state.

**Review by huan-Mongo — CHANGES_REQUESTED** [review]
Oh, I am able to see the rendering now, strange. 

Anyway, I found an existing error on the `COMMITTED` state entry and there is a mistake in my ticket description as well.

File: source/reference/mongosync-states.txt (hunk 2 above)
Comment (kennethdyer) [reply source/reference/mongosync-states.txt:86]: Fixed

File: source/reference/mongosync-states.txt (hunk 3 above)
Comment (kennethdyer) [reply source/reference/mongosync-states.txt:97]: Fixed

**Comment by kennethdyer** [comment]
@huan-Mongo, Changes applied.  Anything else you'd like to see?

**Comment by kennethdyer** [comment]
@huan-Mongo, Good catch. Fixed now.

**Review by huan-Mongo — APPROVED** [review]
LGTM! Thanks for the change.

**Pull request merged**